Formula/portable-openssl.rb

```diff
@@ -28,8 +28,8 @@ class PortableOpenssl < PortableFormula
 
   resource "cacert" do
     # https://curl.se/docs/caextract.html
-    url "https://curl.se/ca/cacert-2024-07-02.pem"
-    sha256 "1bf458412568e134a4514f5e170a328d11091e071c7110955c9884ed87972ac9"
+    url "https://curl.se/ca/cacert-2024-09-24.pem"
+    sha256 "189d3cf6d103185fba06d76c1af915263c6d42225481a1759e853b33ac857540"
```

Comment:
I feel like we should be able to make this use `ca-certificates` in Homebrew/core.

There's a bootstrap problem, but perhaps it might make sense to have a minimal set of CA certs to bootstrap with (that won't need updating quite as regularly) so that it then uses `ca-certificates` from Homebrew/core thereafter.

Reply:
We don't want to import old system certs like `ca-certificates` does. What we ship here ships to all whereas `ca-certificates` is personalised.

Reply:
> So we'd basically need `ca-certificates` without the post install run (i.e. direct from bottle tarball).

I think that should be available at `Formula["ca-certificates"].opt_pkgshare/"cacert.pem"`